HDIClusterConfigurationManagement/src/main/java/com/microsoft/configuration/management/Main.java

```diff
@@ -53,8 +76,6 @@ public static void main(String[] args) throws  Exception {
         String ambariPassword = configuration.get("ambariPassword").getAsString();
         String clusterName = ambariUrl.getHost().split("\\.")[0];
 
-        SkipServerCertificateValidation();
```

Comment:
We will update the argument passing to be command line instead of interactive and enable this again.